great_expectations/rule_based_profiler/helpers/cardinality_checker.py

```diff
@@ -79,6 +79,7 @@ class CardinalityLimitMode(enum.Enum):
     REL_25 = RelativeCardinalityLimit("REL_25", 0.25)
     REL_50 = RelativeCardinalityLimit("REL_50", 0.50)
     REL_75 = RelativeCardinalityLimit("REL_75", 0.75)
+    REL_100 = RelativeCardinalityLimit("REL_100", 1.0)
```

Comment:
I really like this change, and think the resulting ExpectationSuite is going to be a LOT cleaner. Thank you for this @talagluck and @alexsherstinsky and @anthonyburdi